packages/core/admin/admin/src/components/UnstableGuidedTour/Tours.tsx

```diff
@@ -117,9 +178,15 @@ const UnstableGuidedTourTooltip = ({
     };
   }, [isPopoverOpen]);
 
+  const Step = React.useMemo(() => createStepComponents(tourName), [tourName]);
+
   return (
     <>
-      {isPopoverOpen && <GuidedTourOverlay />}
+      {isPopoverOpen && (
+        <Portal>
```

Comment:
Full transparency: if I do this inside the Popover.Content to share it's portal for some reason the overlay is rendered twice and therefore is twice as dark 🤦‍♂️. I don't dig duplicating the condition but also not thaaat bothered by it.